packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -46,6 +48,8 @@
     "removeMenuLabel": "Remove",
     "returnToCallButtonAriaDescription": "Return to Call",
     "returnToCallButtonAriaLabel": "Back",
+    "roomNotFoundDetails": "Room ID provided in call locator is not valid.",
```

Comment:
The end user doesn't know anything about "call locator". Let's double check on these strings with UX.